assets/javascripts/discourse/connectors/composer-fields/persona-selector.js

```diff
@@ -0,0 +1,53 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+
+function isBotMessage(composer, currentUser) {
+  if (
+    composer &&
+    composer.targetRecipients &&
+    currentUser.ai_enabled_chat_bots
+  ) {
+    let reciepients = composer.targetRecipients.split(",");
```

Comment:
nit: should be `const` ... I swear we had an eslint for this at some point, now it's gone. Maybe I am nuts